### ISSUE TITLE
iostat/sar - kB/s MB/s should be kiB/s and MiB/s

### ISSUE THREAD

**Issue by Pitterling** [sysstat/sysstat#313]
as units are transformed by 1024 they should be correctly labeled according to IEC

https://simple.wikipedia.org/wiki/Kilobyte
https://simple.wikipedia.org/wiki/Megabyte
https://simple.wikipedia.org/wiki/Mebibyte

**Comment by sysstat** [comment]
Yep, you're not wrong... But I'm afraid that modifying fields names used by sysstat could break third-party programs or scripts that parse sysstat commands' output.
Anyway things have been made clear in sysstat man pages where we say that we actually use kibibytes, mebibytes, etc.

**Comment by junka** [comment]
I opened an issue related to this. Maybe we can change the values instead of the labels?
https://github.com/sysstat/sysstat/issues/316

**Comment by sysstat** [comment]
No, changing the values would also lead to confusion. No good solution for something which has been there for years.
I noticed that sysstat's FAQ was mute on this. I should probably add an entry about it...

**Comment by Pitterling** [comment]
You could add 2 new options providing the correct label/value outputs.
